docs/excel/excel-add-ins-events.md

```diff
@@ -133,27 +133,27 @@ function remove() {
 > [!NOTE]
 > This feature is currently available only in public preview (beta). To use it, you must reference the beta library of the Office.js CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js.
 
-Events are turned on and off at the [runtime](https://docs.microsoft.com/en-us/javascript/api/excel/excel.runtime?view=office-js) level. 
+Some scenarios benefit from the performance boost given by turning off events. 
```

Comment:
Suggest that you use "disabling events" instead of "turning off events" (to match the lingo used in the section heading).  

Also, you might consider rewording to something more along the lines of: "In some scenarios, performance of an add-in may be improved by disabling events."